rasa/cli/arguments/train.py

```diff
@@ -21,52 +24,63 @@ def set_train_arguments(parser: argparse.ArgumentParser):
     add_augmentation_param(parser)
     add_debug_plots_param(parser)
 
-    add_num_threads_param(parser)
+    _add_num_threads_param(parser)
 
-    add_model_name_param(parser)
+    _add_model_name_param(parser)
     add_persist_nlu_data_param(parser)
     add_force_param(parser)
+    add_finetune_params(parser)
 
 
-def set_train_core_arguments(parser: argparse.ArgumentParser):
+def set_train_core_arguments(parser: argparse.ArgumentParser) -> None:
+    """Specifies CLI arguments for `rasa train core`."""
     add_stories_param(parser)
     add_domain_param(parser)
-    add_core_config_param(parser)
+    _add_core_config_param(parser)
     add_out_param(parser, help_text="Directory where your models should be stored.")
 
     add_augmentation_param(parser)
     add_debug_plots_param(parser)
 
     add_force_param(parser)
 
-    add_model_name_param(parser)
+    _add_model_name_param(parser)
 
     compare_arguments = parser.add_argument_group("Comparison Arguments")
-    add_compare_params(compare_arguments)
+    _add_compare_params(compare_arguments)
+    add_finetune_params(parser)
 
 
-def set_train_nlu_arguments(parser: argparse.ArgumentParser):
+def set_train_nlu_arguments(parser: argparse.ArgumentParser) -> None:
+    """Specifies CLI arguments for `rasa train nlu`."""
     add_config_param(parser)
     add_domain_param(parser, default=None)
     add_out_param(parser, help_text="Directory where your models should be stored.")
 
     add_nlu_data_param(parser, help_text="File or folder containing your NLU data.")
 
-    add_num_threads_param(parser)
+    _add_num_threads_param(parser)
 
-    add_model_name_param(parser)
+    _add_model_name_param(parser)
     add_persist_nlu_data_param(parser)
+    add_finetune_params(parser)
 
 
-def add_force_param(parser: Union[argparse.ArgumentParser, argparse._ActionsContainer]):
+def add_force_param(
+    parser: Union[argparse.ArgumentParser, argparse._ActionsContainer]
+) -> None:
+    """Specifies if the model should be trained from scratch."""
```

Comment:
`Args` section is missing in the docstring but that's probably something CI will complain about.

Reply:
It's missing on purpose because I don't see any value in having full docstrings for these helper functions. The CI actually allows one-line docstrings 😁 What do you think?